kornia/augmentation/augmentation.py

```diff
@@ -200,10 +235,24 @@ def generate_parameters(self, batch_shape: torch.Size) -> Dict[str, torch.Tensor
             self.device, self.dtype)
 
     def compute_transformation(self, input: torch.Tensor, params: Dict[str, torch.Tensor]) -> torch.Tensor:
-        return F.compute_intensity_transformation(input)
+        return self.identity_matrix(input)
 
-    def apply_transform(self, input: torch.Tensor, params: Dict[str, torch.Tensor]) -> torch.Tensor:
-        return F.apply_color_jitter(input, params)
+    def apply_transform(
+        self, input: torch.Tensor, params: Dict[str, torch.Tensor], transform: Optional[torch.Tensor] = None
+    ) -> torch.Tensor:
+        transforms = [
+            lambda img: adjust_brightness(img, params['brightness_factor'].to(img.dtype) - 1),
```

Comment:
```suggestion
            lambda img: adjust_brightness(img, params['brightness_factor'].to(img) - 1),
```

Reply:
same for below

Reply:
I removed it. It was handled inside the function. Actually, we shall refactor the kornia core to avoid using ```to``` for type conversions to let it crash.